dune-project

```diff
@@ -11,15 +11,15 @@
 
 (maintainers "Paul-Elliot <peada@free.fr>" "Jules Aguillon <jules@j3s.fr>")
 
-(license LICENSE)
+(license MIT)
 
 (documentation https://url/to/documentation)
 
 (package
```

Comment:
At this point, I think we should correctly fill the dune project. I don't think external contributors would be confident in doing that for us.